### DISCUSSION TITLE
Test 'DiaFlux:Abnormality' does not exist

### DISCUSSION THREAD

**Discussion by VitaliyMez** [denkbares/d3web-KnowWE#17] [Q&A]
Hi, I try to apply the Test 'DiaFlux:Abnormality'

@test: "DiaFlux:Abnormality" expert_sys «message»

Have: "Test 'DiaFlux:Abnormality' does not exist"

Please help.

**Comment by AlbrechtStriffler** [comment]
Hi Vitaliy,
that test seems to be closed source. It is contained in the build running d3web.de, but not the downloadable distribution package. Are you sure u need it? I remember it to be useful in a really project specific context...
If you explain your use case, maybe I can help in some other way.
Regards
Albrecht
